app/js/views/services.jsx

```diff
@@ -65,25 +63,15 @@ export default class Services extends React.Component {
     this.setState({ services });
   }
 
-  dismissModal() {
-    this.setState({ isModalVisible: false });
+  renderHeader() {
+    return super.renderHeader('My Home');
   }
 
-  render() {
+  renderBody() {
     return (
-      <div className="app-view">
-        <header className="app-view__header">
-          <h1>My Home</h1>
-        </header>
-        <section className="app-view__body">
-          <h2>General</h2>
-          <ServicesList services={this.state.services} foxbox={this.foxbox}/>
-        </section>
-        <footer className="app-view__footer">
-          <NavigationMenu foxbox={this.foxbox}/>
-        </footer>
-        <Modal visible={this.state.isModalVisible} title={this.state.title}
-               body={this.state.body} dismiss={this.dismissModal.bind(this)}/>
+      <div className="app-view__body-placeholder">
```

Comment:
I don't like it as much as you do :) But here is my rationale:

I want main class (```app-view__body```) to be defined in base view as it's base view responsibility to define view structure including "main" css classes, but then every view should return single element in ```renderBody``` (because of [1]), so ```app-view__body-placeholder``` is not a requirement, but rather an optional thing - you use it only if you want your wrapper node to fill entire ```app-view__body``` section. I can think of two alternatives:
* Rename ```app-view__body-placeholder``` to something more obvious like ```app-view_fill-body``` or something to stress that it's styling only and doesn't have any functional meaning;
* Remove this class and if view wants its body to fill entire ```app-view__body``` it defines its own styling;
* Or we can force it with something ```app-view__body > :first-child``` - but I doooon't like this option :) It's not BEM'y, not performant and make a fixed decision on behalf of the particular view itself.

What do you think @gmarty ?

[1] http://facebook.github.io/react/tips/maximum-number-of-jsx-root-nodes.html